lib/reservation_tracker.rb

```diff
@@ -0,0 +1,108 @@
+require 'pry'
+require 'date'
+require_relative 'room'
+require_relative 'block'
+require_relative 'reservation'
+# Keeps track of the list of reservations
+# module Hotel
+class ReservationTracker
+  attr_reader :rooms, :check_in_date, :check_out_date, :reservations, :unreserved_rooms, :blocks
+
+  def initialize
+    @rooms = create_rooms
+    @reservations = []
+    @unreserved_rooms = []
```

Comment:
Does this instance variable `@unreserved_rooms` ever get used? If it doesn't get used, can we get rid of it?